Sources/LiveViewNative/Modifiers/View Configuration Modifiers/DraggableModifier.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  DraggableModifier.swift
+//  
+//
+//  Created by murtza on 30/05/2023.
+//
+
+import SwiftUI
+
+/// Activates this view as the source of a drag and drop operation.
+///
+/// ```html
+/// <Text modifiers={draggable(@native, payload: "ABC", preview: :my_preview)}>
+///     ABC
+///     <Image system-name="heart.fill" template={:my_preview}>
+/// </Text>
+/// ```
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct DraggableModifier<R: RootRegistry>: ViewModifier, Decodable {
```

Comment:
Please add `@available` here so the watchOS unavailability is clear in the docs.